homeassistant/components/ezviz/sensor.py

```diff
@@ -18,25 +18,64 @@
 PARALLEL_UPDATES = 1
 
 SENSOR_TYPES: dict[str, SensorEntityDescription] = {
-    "sw_version": SensorEntityDescription(key="sw_version"),
     "battery_level": SensorEntityDescription(
         key="battery_level",
+        name="Battery",
```

Comment:
We don't have to provide a name when a translation key is set. (This applies to all other cases in this PR too).